app/src/main/java/com/google/samples/apps/sunflower/viewmodels/PlantItemViewModel.kt

```diff
@@ -0,0 +1,19 @@
+package com.google.samples.apps.sunflower.viewmodels
+
+import android.view.View
+import androidx.lifecycle.ViewModel
+import androidx.navigation.findNavController
+import com.google.samples.apps.sunflower.PlantListFragmentDirections
+import com.google.samples.apps.sunflower.data.Plant
+
+class PlantItemViewModel(
+    private val plant: Plant
```

Comment:
Let public val is ok, no need to import a extra function to do same thing.

Reply:
ok, changed